### PR TITLE
Merge appsembler/master into appsembler/feature/mergeDogwood.3

### PR DESCRIPTION
**Pull request by melvinsoft**
We need to introduce some changes into our `feature/mergeDogwood.3`, I know we are moving off this branching model, but some customers like GAP this to have the search fixes working ASAP.

### CODE REVIEW

**Comment by melvinsoft** [comment]
[Gap: Merge fix for properly hiding courses via Advanced Settings](https://trello.com/c/ZAUhIDE5/959-gap-merge-fix-for-properly-hiding-courses-via-advanced-settings)

File: lms/envs/aws_appsembler.py [hunk 1]
```diff
@@ -11,3 +11,16 @@
...
 COURSE_CATALOG_VISIBILITY_PERMISSION = 'see_in_catalog'
 COURSE_ABOUT_VISIBILITY_PERMISSION = 'see_about_page'
+
+# SENTRY
+SENTRY_DSN = AUTH_TOKENS.get('SENTRY_DSN', False)
+
+if SENTRY_DSN:
+
+    # Set your DSN value
+    RAVEN_CONFIG = {
```
Comment (johnbaldwin) [inline lms/envs/aws_appsembler.py]: IS `RAVEN_CONFIG` declared anywhere else? or not used if not `SENTRY_DSN`?

**Comment by johnbaldwin** [comment]
Just to note, I'm not familiar with Raven or most of the settings related to this PR, but it all looks reasonable to me. Just had one question

File: lms/envs/aws_appsembler.py (hunk 1 above)
Comment (melvinsoft) [inline lms/envs/aws_appsembler.py]: @johnbaldwin good question, I don't know really, probably @mrgnr has more insights.

**Pull request merged**